plugin.rb

```diff
@@ -55,6 +55,15 @@
     end
 
     any_user_agent[:disallow] ||= []
-    any_user_agent[:disallow] << "/docs/"
+    any_user_agent[:disallow] << "/#{SiteSetting.docs_url_path}/"
+  end
+
+  on(:site_setting_changed) do |name, old_value, new_value|
+    next if name != :docs_url_path
+
+    Discourse::Application.routes.append do
```

Comment:
is this old code? feels like it is not safe

Reply:
Oh yes, that's old; will remove it.